modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -125,7 +133,7 @@ def to_numpy(self, **kwargs):
         Parameters
         ----------
         **kwargs : dict
-            Additional keyword arguments to be passed in `to_numpy`.
+            Additional keyword arguments to be passed in ``to_numpy``.
```

Comment:
There was a discussion earlier when we decided to use one back stick for methods and parameters, two for objects and classes.

Reply:
I didn't know, just copied from [another docstring](https://github.com/modin-project/modin/pull/3991/files#diff-9982d8740602ab4ffe8104fb6590b98d10b065e8731fa2cc4b286bc1e84c20d5L227). 

The next two files: `dataframe_supported.rst` and `series_supported.rst` also use two back stick for methods. I think that this needs to be fixed at once in all the code so that this does not happen again.

Reply:
Those files should be formatted/extended/improved as part of our activity on docs.